src/usr/lib/rsetup/tui/task/task.sh

```diff
@@ -3,10 +3,12 @@
 # shellcheck source=src/usr/lib/rsetup/tui/task/docker/docker.sh
 source "/usr/lib/rsetup/tui/task/docker/docker.sh"
 source "/usr/lib/rsetup/tui/task/ssh/ssh.sh"
+source "/usr/lib/rsetup/tui/task/baota/baota.sh"
 
 __task() {
     menu_init
     menu_add __task_docker          "Docker"
     menu_add __task_ssh             "SSH"
```

Comment:
```suggestion
    if $DEBUG
    then
        menu_add __task_docker          "Docker"
        menu_add __task_ssh             "SSH"
    fi
```
Those 2 will need to be reworked. Hide them for now.